docs/tutorial/part-zero/index.md

```diff
@@ -1,235 +1,235 @@
 ---
-title: Set Up Your Development Environment
+title: Imposta il tuo Ambiente di Sviluppo
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Before you start building your first Gatsby site, you’ll need to familiarize yourself with some core web technologies and make sure that you have installed all required software tools.
+Prima di iniziare a creare il tuo primo sito Gatsby, avrai bisogno di familiarizzare con alcune tecnologie web di base e assicurarti di aver installato tutti gli strumenti software richiesti.
 
-## Familiarize yourself with the command line
+## Familiarizzare con la linea di comando
 
-The command line is a text-based interface used to run commands on your computer. You’ll also often see it referred to as the terminal. In this tutorial, we’ll use both interchangeably. It’s a lot like using the Finder on a Mac or Explorer on Windows. Finder and Explorer are examples of graphical user interfaces (GUI). The command line is a powerful, text-based way to interact with your computer.
+La linea di comando è una interfaccia testuale che esegue comandi sul tuo computer. Sarà anche fatto riferimento ad essa come terminale. In questo tutorial, useremo entrambi. È un po come usare il Finder sul Mac o Explorer su Windows. Il Finder e Explorer sono esempi di interfaccia grafica (GUI). La linea di comando è un modo potente, basato sul testo, per interagire con il computer.
```

Comment:
Formulazione un po' contorta, la rivedresti?

Po' vuole l'apostrofo

Reply:
Ho cambiato un po' prendendo spunto da wikipedia. Usando riga invece di linea
https://it.wikipedia.org/wiki/Interfaccia_a_riga_di_comando